packages/core/src/modules/proofs/protocol/v1/__tests__/indy-proof-presentation.test.ts

```diff
@@ -1,28 +1,28 @@
 import type { Agent } from '../../../../../agent/Agent'
 import type { ConnectionRecord } from '../../../../connections/repository/ConnectionRecord'
+import type { ProofExchangeRecord } from '../../../repository'
 import type { PresentationPreview } from '../models/V1PresentationPreview'
 
-import { setupProofsTest, waitForProofExchangeRecord } from '../../../../../../tests/helpers'
+import { setupIndyProofsTest, waitForProofExchangeRecord } from '../../../../../../tests/helpers'
 import testLogger from '../../../../../../tests/logger'
 import { DidCommMessageRepository } from '../../../../../storage/didcomm'
 import { ProofState } from '../../../models/ProofState'
-import { ProofExchangeRecord } from '../../../repository/ProofExchangeRecord'
 import { V1PresentationMessage, V1ProposePresentationMessage, V1RequestPresentationMessage } from '../messages'
 
 describe('Present Proof', () => {
   let faberAgent: Agent
   let aliceAgent: Agent
   let aliceConnection: ConnectionRecord
   let presentationPreview: PresentationPreview
-  let faberProofExchangeRecord: ProofExchangeRecord
-  let aliceProofExchangeRecord: ProofExchangeRecord
+  let faberProofRecord: ProofExchangeRecord
```

Comment:
Same as before: faberProofRecord was changed to 'faberProofExchangeRecord' and here is again with that older name.

Reply:
Yes I think it was just a question of me not merging everything down to this branch correctly. Good spot.